ils-A-analog.py

```diff
@@ -0,0 +1,26 @@
+from irods.test.helpers import make_session
+import  irods.exception
+import sys
+import getopt
+
+s=make_session()
+
+def get(p,ses=s):
+    try:
+        return ses.collections.get(p)
+    except irods.exception.CollectionDoesNotExist:
+        return ses.data_objects.get(p)
```

Comment:
Does this mean the PRC doesn't provide functions like `is_collection(path)` and `is_data_object(path)`?

Reply:
I'm almost certain it does not. Should it provide those functions as utilities?  It would be easy to do.

Reply:
I think it would be good to have those. Then we wouldn't have to rely on exceptions for this kind of thing.

Reply:
This script has been deleted from the PR, and #412 has been made for including commonly needed utility functions such as `is_collection,` etc.